landoapi/api/transplants.py

```diff
@@ -251,6 +271,14 @@ def post(data):
             f"{to_land}, {landing_repo}, {stack_data}"
         )
 
+    if set(flags) - set(landing_repo.commit_flags):
+        raise ProblemException(
+            400,
+            "Invalid flags specified",
+            f"Flags must be one or more of {landing_repo.commit_flags}; {flags} provided.",
```

Comment:
Nit: if we save the result of `set(flags) - set(landing_repo.commit_flags)` we could explicitly return the bad flag instead of returning all the passed flags.

Reply:
Done.